File: .py/Program/PetShop.py
class PetShop:
    __ID = 0
    __nama = ""
    __harga = 0
    __stok = 0

    def __init__(self, ID=0, nama="", harga=0, stok=0):
        self.__ID = ID
        self.__nama = nama
        self.__harga = harga
        self.__stok = stok

    def setID(self, ID):
        self.__ID = ID

    def getID(self):
        return self.__ID

    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama

    def setHarga(self, harga):
        self.__harga = harga

    def getHarga(self):
        return self.__harga

    def setStok(self, stok):
        self.__stok = stok

    def getStok(self):
        return self.__stok

File: .php/Program/PetShop.php
<?php
class PetShop {
    private $ID = 0;
    private $nama = '';
    private $stok = 0;
    private $harga = 0;
    private $foto = '';

    public function __construct($ID = 0, $nama = '', $stok = 0, $harga = 0, $foto = '') {
        $this->ID = $ID;
        $this->nama = $nama;
        $this->stok = $stok;
        $this->harga = $harga;
        $this->foto = $foto;
    }

    public function setID ($ID) { $this->ID = $ID; }
    public function getID () { return $this->ID; }

    public function setNama ($nama) { $this->nama = $nama; }
    public function getNama () { return $this->nama; }

    public function setStok ($stok) { $this->stok = $stok; }
    public function getStok () { return $this->stok; }

    public function setHarga ($harga) { $this->harga = $harga; }
    public function getHarga () { return $this->harga; }

    public function setFoto ($foto) { $this->foto = $foto; }
    public function getFoto () { return $this->foto; }
}

?>